docs/designers-developers/key-concepts.md

```diff
@@ -143,4 +143,4 @@ A purely dynamic block that is to be server rendered before display could look l
 
 ## The Gutenberg Lifecycle
 
-In summary, the workflow for editing a Gutenberg post starts with taking the persisted version of the document and generating the in-memory tree, aided by the presence of token delimiters. It ends with the reverse: serialization of blocks into `post_content`. During editing, all manipulations happen within the block tree. In summary, a Gutenberg post is built upon an in-memory data structure which gets persisted somehow in an fully-isomorphic way. Right now that persistence is via a serialization/parser pair but could just as easily be replaced through a plugin to store the data structure as a JSON blob somewhere else.
+In summary, the workflow for editing a Gutenberg post starts with using token delimeters to generate an in-memory tree from the persisted version of the document. The process ends by reversing the serialization of blocks into the `post_content`. During editing, all manipulations happen within the block tree. Alternatively, a Gutenberg post is built and manipulated using an isomorphically persistied, in-memory data structure. Although Gutenberg post persistence is currently achieved using a serialization/parser pair, in hypothetical scenarios the post data structure could also be stored using a plugin or retreived from a remote JSON file.
```

Comment:
>The process ends by reversing the serialization of blocks

The phrasing here seems to imply that a serialization is reversed, which doesn't really make sense. The previous phrasing was more to emphasize the _inverse_ of the initial parse, i.e. completing the cycle of "Markup -> (Parse) -> Blocks -> (Serialize) -> Markup".